src/apps/service/apps/de/jobs/common.clj

```diff
@@ -114,6 +114,16 @@
        (take-while (comp nil? :external_app_id))
        (reduce #(.buildStep request-builder %1 %2) [])))
 
+(defn load-htcondor-extra-requirements
+  [app-id]
+  (select :apps_htcondor_extra
+          (fields [:extra_requirements])
+          (where {:app_id app-id})))
```

Comment:
... I've already realized I probably need to add `(first (first ))` or something here